src/MailTracking.php

```diff
@@ -90,6 +87,38 @@ public function recordMail(Swift_Message $email)
     }
 
     /**
+     * Assert that the last email was bcc'ed to the given address.
+     *
+     * @param string        $bcc
+     * @param Swift_Message $message
+     *
+     * @return PHPUnit_Framework_TestCase $this
+     */
+    protected function seeEmailBcc($bcc, Swift_Message $message = null)
+    {
+        $this->assertArrayHasKey($bcc, (array)$this->getEmail($message)
+                                                   ->getBcc(), "No email was bcc'ed to $bcc.");
+
+        return $this;
+    }
+
+    /**
+     * Assert that the last email was cc'ed to the given address.
+     *
+     * @param string        $cc
+     * @param Swift_Message $message
+     *
+     * @return PHPUnit_Framework_TestCase $this
+     */
+    protected function seeEmailCc($cc, Swift_Message $message = null)
+    {
+        $this->assertArrayHasKey($cc, (array)$this->getEmail($message)
+                                                  ->getCc(), "No email was cc'ed to $cc.");
+
+        return $this;
+    }
```

Comment:
I made a separate issue for that #18 so I want to do that in a different feature.